src/main/java/org/candlepin/subscriptions/conduit/inventory/kafka/KafkaEnabledInventoryService.java

```diff
@@ -97,6 +102,14 @@ protected void sendHostUpdate(List<ConduitFacts> facts) {
         }
     }
 
+    @Recover
+    protected void recover(Throwable throwable) {
+        log.info("sendHostUpdate recover");
+        log.info(String.format("Exception type: {}", throwable.getClass().getName()));
+        log.info(String.format("Exception message: {}", throwable.getMessage()));
+    }
+
+
```

Comment:
I'm not a fan of having a recover method that doesn't do anything except log.

Reply:
So, I'd recommend just removing the method.

Reply:
It was part of one of the examples I found.  I'll remove it.  Doing so will also solve the Sonar Cloud errors I'm getting.

Reply:
Turns out, I can't remove it.  The unit test fails if I do.  I will look into this, but if I can't remove it, is there something besides logging that you would like it to do?